SteamKit2/SteamKit2/Steam/CDN/Client.cs

```diff
@@ -230,18 +229,26 @@ public async Task<int> DownloadDepotChunkAsync( uint depotId, DepotManifest.Chun
             var chunkID = Utils.EncodeHexString( chunk.ChunkID );
             var url = $"depot/{depotId}/chunk/{chunkID}";
 
-            using var request = new HttpRequestMessage( HttpMethod.Get, BuildCommand( server, url, cdnAuthToken, proxyServer ) );
-
+            var builtUrl = BuildCommand( server, url, cdnAuthToken, proxyServer );
+            var request = new HttpRequestMessage( HttpMethod.Get, builtUrl );
+            if ( UseLancacheServer )
+            {
+                request = BuildLancacheRequest( server, url, cdnAuthToken );
+            }
+
             using var cts = new CancellationTokenSource();
             cts.CancelAfter( RequestTimeout );
 
             try
             {
-                using var response = await httpClient.SendAsync( request, HttpCompletionOption.ResponseHeadersRead, cts.Token ).ConfigureAwait( false );
+                using var response = await httpClient
```

Comment:
undo the unrelated formatting changes pls

Reply:
Sorry, didn't catch that my editor did that.  Reverted.